css/css-text/white-space/text-wrap-balance-top-to-bottom.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#text-wrap">
+<meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+#container {
+  height: 15ch;
+  writing-mode: vertical-rl;
+}
+.balance {
+  text-wrap: balance;
+}
+</style>
+<div id="container"></div>
+<script>
+const container = document.getElementById('container');
+for (const text of [
+    '平衡应该',
```

Comment:
This seems to fit in a single line within height: 15ch, so balancing won't have an effect on this.

Reply:
Yes it is the intention to test strings that displays as one, two, and three lines. To make sure that balancing will not add the number of lines.